lib/board.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'cell.rb'
+
+class Board
+  attr_reader :grid
+
+  def initialize
+    @grid = create_grid
+  end
+
+  def set_cell(x, y, value = 'X')
+    raise "Cell taken by #{grid[x][y].value}" if grid[x][y].value != ''
```

Comment:
Readability: You have a get_cell method that you could use here to improve the readability.

Encapsulation: Could it be the cell's job to check the status of a cell?